docs/CONTRIBUTING.md

```diff
@@ -171,8 +171,24 @@ docker exec -it student_dashboard_mysql mysql_upgrade \
 When prompted, specify the password for the root MySQL user.
 It should be found in the `MYSQL.ROOT_PASSWORD` property of `env.hjson`.
 
-[Next: Contributors](../docs/CONTRIBUTORS.md)
+### Backing up the development database
+
+To backup the development databaes with the default settings, use this command
+```sh
+docker exec student_dashboard_mysql mysqldump --socket=/tmp/mysql.sock -uroot -pstudent_dashboard_root_pw student_dashboard > student_dashboard.sql
+```
+
+To restore this database to a new database you had created as an admin called student_dashboard_new (CREATE database student_dashboard_new). You can also just restore it to the same database (student_dashboard).
+
+```sh
+docker exec -i student_dashboard_mysql mysql -uroot -pstudent_dashboard_root_pw student_dashboard_new < student_dashboard.sql`
```

Comment:
Since we are restoring data to the same database most of the time, I think we can use student_dashboard instead of student_dashboard_new here as the database name

Reply:
Yeah, makes sense, I just had this as an example for restoring to a new database. I can update the command and leave the comment.

Reply:
```suggestion
docker exec -i student_dashboard_mysql mysql -uroot -pstudent_dashboard_root_pw student_dashboard < student_dashboard.sql`
```